app/api/v1/endpoints/websocket.py

```
@@ -0,0 +1,28 @@
+"""
+WebSocket endpoints for real-time updates
+"""
+from fastapi import APIRouter, WebSocket, Depends
+from app.core.websocket import manager
+from app.api.v1.endpoints.auth import get_current_user
+from app.models.user import User
+
+router = APIRouter()
+
+@router.websocket("/ws")
+async def websocket_endpoint(websocket: WebSocket, user: User = Depends(get_current_user)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)